OCPP-J/src/main/java/eu/chargetime/ocpp/WebSocketListener.java

```diff
@@ -55,6 +59,7 @@ of this software and associated documentation files (the "Software"), to deal
   private final Map<WebSocket, WebSocketReceiver> sockets;
   private volatile boolean closed = true;
   private boolean handleRequestAsync;
+  private CompletableFuture<Void> socketBound;
```

Comment:
nitpick: convert to local variable since only used there

Reply:
Done;